friends/src/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { Provider } from 'react-redux';
+import { applyMiddleware, compose, createStore } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+
+import { addTokenToLocalStorage } from './utilities/addTokenToLocalStorage';
+import rootReducer from './reducers/reducers';
+
+import App from './App';
+import './main.css';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
consider using the redux-devtools-extension package instead of this, it's still a bit of a lengthy name but may be easier to remember than this (definitely looks cleaner though): https://github.com/zalmoxisus/redux-devtools-extension